Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -794,7 +794,7 @@
 #endif
 
 #if (!defined(ENABLE_APP_HIGHLIGHTS) \
-    && ((PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 120000) \
+    && (PLATFORM(MAC) \
         || (PLATFORM(IOS) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 150000) \
```

Comment:
Don't need this check either (there are more iOS version checks below, which I won't comment on).

Reply:
Yep, I'll do these in a follow-up.